options.go

```diff
@@ -0,0 +1,85 @@
+package jrpc
+
+import (
+	"net/http"
+	"time"
+)
+
+// Option func type
+type Option func(s *Server)
+
+// Auth sets basic auth credentials, required
+func Auth(user, password string) Option {
+	return func(s *Server) {
+		s.authUser = user
+		s.authPasswd = password
+	}
+}
+
+// AppName sets custom application name for server signature, optional
+func AppName(name string) Option {
+	return func(s *Server) {
+		s.appName = name
+	}
+}
+
+// Version sets app version value for server signature, optional
+func Version(version string) Option {
+	return func(s *Server) {
+		s.version = version
+	}
+}
+
+// Timeouts sets server timeout values such as ReadHeader, Write and Idle timeout, optional.
+// If this option not defined server use default timeout values
+func Timeouts(limits Limits) Option {
```

Comment:
I think, it will be better to rename Limits here to Timeouts to avoid confusions, as we have other limits already

Reply:
Should is stay type as `Limits` or define new type (e.g. `Timeouts`) with fields for timeouts only?